JS-Algorithms/challenges.js

```diff
@@ -71,12 +88,23 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 
 const ownPower = (number, lastDigits) => {
 	// YOUR CODE HERE...
+	let myResult = 0;
+	let count = 1;
```

Comment:
What does this variable `count` do? It seems you're only adding "1" on each iteration but doing nothing with it